qiling/core.py

```diff
@@ -97,7 +97,8 @@ def __init__(
             stack_address=0,
             stack_size=0,
             interp_base=0,
-            automatize_input=False
+            automatize_input=False,
+            windows_config=None
```

Comment:
automatized and config remove from here. this two var i assume happens during ql.run so we can just call ql.xxx before ql.run()

Reply:
Yes, i can remove them from here, but how can the user decide if he wants to automatize the input and the config file otherwise?